### ISSUE TITLE
Accessibility > Menu > Tab list > menu for overflow tabs in tab list

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluent-ui-react#691]
### Problem description
As output during the implementation
https://github.com/stardust-ui/react/pull/686

We figure out that we are missing accessibility for overflow tabs, which can be displayed in the menu. 

### Proposed solution
Need to find out the proper way how to handle it. 
There need to be element which will be holding menu to trigger, an it can be:
- part of the tab list 
        - easy to discover in tab list
        - challenge how to tell user there is menu/popup
- outside the tab list (following tab list as next TAB stop) 
        - easy to tell user there is menu/popup
        - maybe harder do discover for user 
### MVP
[Fork, update, and replace this pen if you can show the proposed solution]:
https://codesandbox.io/s/2l3n74j9y
